packaging/rpm/microshift.spec

```diff
@@ -519,6 +519,9 @@ fi
 # Use Git command to generate the log and replace the VERSION string
 # LANG=C git log --date="format:%a %b %d %Y" --pretty="tformat:* %cd %an <%ae> VERSION%n- %s%n" packaging/rpm/microshift.spec
 %changelog
+* Mon Aug 26 2024 Nadia Pinaeva <n.m.pinaeva@gmail.com> 4.18.0
```

Comment:
Are we planning to backport this fix to 4.17.0? If so, let's change the version to 4.17.0 here so that we can cherrypick the PR.